### PR TITLE
GF 21-30

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: great-mystery-great-whore/original/025-chapter-24.adoc [hunk 1]
```diff
@@ -278,8 +278,7 @@ or steeple-houses to vent their doctrine,`' etc.
 
 [.discourse-part]
 A+++.+++ These are the marks they give to us,
-and to prove us according to the title of their book,
-'`Perfect Pharisees`'. The apostles preached as they went up and down, in towns, markets,
+and to prove us according to the title of their book, [.book-title]#Perfect Pharisees.# The apostles preached as they went up and down, in towns, markets,
```
Comment (maijapapaya) [inline great-mystery-great-whore/original/025-chapter-24.adoc]: I wasn't sure if I should make this into a book title or not.

Comment (jaredh159) [reply great-mystery-great-whore/original/025-chapter-24.adoc]: Yeah, a bit confusing... I think I lean towards keeping it single quotes how it was before, now that I've thought about it a bit.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Looks great, just the one change mentioned. thanks!

**Comment by jaredh159** [comment]
thanks! i fixed the one lint error.

**Pull request merged**